src/email-templates/new_company_application_admins.handlebars

```diff
@@ -0,0 +1,10 @@
+<h3>A company has requested access to NIJobs</h3>
+<br>
+<p>{{companyName}} <{{email}}> has requested to participate in NIJobs. Below is their motivation:</p>
+<blockquote><i>{{motivation}}</i></blockquote>
```

Comment:
Any reason to use the `<i>` here? [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/i) seems to suggest it would not make much sense.

Reply:
It renders the text as italic. I think it helps to differentiate the text better (the blockquote simply indents it)